share/ruby-install/package_manager.sh

```diff
@@ -3,6 +3,16 @@
 #
 # Auto-detect the package manager.
 #
+
+brew_owner="$(/usr/bin/stat -f %Su "$(command -v brew)")";
```

Comment:
Also, this code should probably go inside of the `detect_package_manager` function, under the `elif command -v brew` branch. No point in running this code if brew isn't installed.